system-container/run-system-container.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+export ANSIBLE_LOG_PATH=/var/log/ansible.log
+exec ansible-playbook -i /etc/ansible/hosts ${OPTS} ${PLAYBOOK_FILE}
```

Comment:
The current image is based in playbook2image and it already provides a wrapper around `ansible-playbook` (the `run` script in the CMD) that handles things like running as non-root and more flexible inventory handling. Do we need a different command here? (I'm curious why...)